iofog-agent-daemon/src/main/java/org/eclipse/iofog/tracking/TrackingInfoUtils.java

```diff
@@ -18,7 +18,7 @@ public static JsonObject getStartTrackingInfo() {
         try {
             String gpsCoordinates = Configuration.getGpsCoordinates();
             String gpsMode = Configuration.getGpsMode().name();
-            boolean developerMode = Configuration.isDeveloperMode();
+            boolean developerMode = Configuration.isSecureMode();
```

Comment:
Should this be the opposite? As in, `developerMode` is the opposite of `secureMode`, or am I not understanding what secure mode means?

Reply:
We are using `secure_mode` for checking certs. If it’s on then we are making https calls. On the other hand we are using `dev_mode` for sending exceptions to sentry or not.